util/CMakeLists.txt

```diff
@@ -135,7 +135,7 @@ if (BUILD_WITH_LDAP)
 endif (BUILD_WITH_LDAP)
 
 if (CMAKE_C_COMPILER_ID STREQUAL "GNU")
```

Comment:
```suggestion
if (CMAKE_C_COMPILER_ID STREQUAL "GNU")
# set C standard for older GNU compilers (GCC >= 6.3.0)
```

Reply:
You mean < 6.3.0?

Reply:
> So with this patch it will work with gcc up to 6.3.0

I guess yes. I misinterpreted your comment. So for gcc >= 6.3.0 you need this patch?

Reply:
No it looks like it is only needed lower than 6.3.
I have added an version check for it,